 02-el.c-issetugid.diff by Gergely Nagy <algernon@debian.org>

 issetugid(2) is not implemented on GNU systems, so #ifdef it out

--- a/libedit/el.c	2008-04-06 02:53:28.000000000 +1100
+++ b/libedit/el.c	2009-06-23 08:51:48.000000000 +1000
@@ -479,12 +479,14 @@ el_source(EditLine *el, const char *fnam
 
 	fp = NULL;
 	if (fname == NULL) {
-#ifdef HAVE_ISSETUGID
+#if 1
 		static const char elpath[] = "/.editrc";
 		char path[MAXPATHLEN];
 
+#ifndef __GLIBC__
 		if (issetugid())
 			return (-1);
+#endif
 		if ((ptr = getenv("HOME")) == NULL)
 			return (-1);
 		if (strlcpy(path, ptr, sizeof(path)) >= sizeof(path))
